Runtime/Tabs/DynamicTab.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using UnityEngine;
+
+namespace Innoactive.Creator.Core.Tabs
+{
+    /// <summary>
+    /// This <inheritdoc cref="ITab"/> implementation defines <seealso cref="GetValue"/> and <seealso cref="SetValue"/> implementation with delegates through the constructor.
+    /// </summary>
+    internal class DynamicTab : ITab
+    {
+        public GUIContent Label { get; }
+
+        public object GetValue()
+        {
+            return getter();
+        }
+
+        public void SetValue(object value)
+        {
+            setter(value);
+        }
```

Comment:
Missing docs

Reply:
its internal

Reply:
I agree that is optional but it would be nice.